### ISSUE TITLE
Count Unicode glyphs instead of JS length

### ISSUE THREAD

**Issue by xem** [lionleaf/dwitter#180]
On Twitter, "💩💩" is 2 chars long.
On Dwitter, 💩💩".length == 4.
Can you please count glyphs instead of chars?
Here's a way to count glyphs properly (with ES6):
[..."💩💩"].length == 2

**Comment by iver56** [comment]
http://blog.jonnew.com/posts/poo-dot-length-equals-two

**Comment by delan** [comment]
It’s worth noting that “💩💩” is 2/140 on Twitter, so making this change would make dwitter more like the social network that inspired it.

**Comment by lionleaf** [comment]
One argument against this: by fixing this bug byte-packing into incomprehensible unicode-strings becomes a very efficient technique. I'm a bit afraid of having all the top dweets be even less readable ;)

**Comment by delan** [comment]
Agreed (regardless of your final decision). It’s not like astral code points are banned, but halving the cost of using them would largely encourage techniques that are pretty brute-force.
